test/built-ins/String/prototype/toWellFormed/to-string.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2022 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-String.prototype.isWellFormed
```

Comment:
Why is this in the `toWellFormed` folder?

Reply:
copy-paste error, thanks, will fix